chainio/clients/avsregistry/reader.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/ethereum/go-ethereum"
 	"github.com/ethereum/go-ethereum/accounts/abi/bind"
 	"github.com/ethereum/go-ethereum/common"
+	gethcommon "github.com/ethereum/go-ethereum/common"
```

Comment:
We should use the other import instead (`common`).

Reply:
Fixed